cf-video-overlays.php

```diff
@@ -16,7 +16,10 @@
  * @return string
  */
 function cf_video_overlays_render( $atts, $modal = false ){
-	$form = Caldera_Forms_Forms::get_form( $atts );
+	if( !class_exists( 'Caldera_forms') ) {
+		return false;
+	}
+	$form = Caldera_Forms_Forms::get_form( $atts['id'] );
```

Comment:
This smells bad, is going to need to change for every new provider. I will rewrite function as an abstract class with a YouTube subclass